### ISSUE TITLE
Commands on Huawei switches that need confirmation

### ISSUE THREAD

**Issue by xvegax** [ktbyers/netmiko#3253]
It seems, Netmiko is not able to properly issue commands on Huawei switches that need confirmations like for example this one:

[LAB-7700]interface XGigabitEthernet 2/0/22
[LAB-7700-XGigabitEthernet2/0/22]**clear configuration this**
Warning: All configurations of the interface will be cleared, and its state will be shutdown. Continue? [Y/N] :
Info: Total 6 command(s) executed, 6 successful, 0 failed.
[LAB-7700-XGigabitEthernet2/0/22]

So, commands with confirmations throw out the confirmation lines which end with a ":" and therefore even adding a simple "y" in a send_config_set(config_commands) after the comand "clear configuration this" (or any other command needing a confirmation) will not work.

Here is another example that does not work:

[LAB-7700]**undo vsi vpls-test**
Warning: The specified VSI will be deleted. Continue? [Y/N] :
[LAB-7700]

Basically, all these commands which need a confirmation start with "Warning" and finish with a ":".

Is it possible to implement this (for Huawei devices, at least)?

Or is there another way around? I was told to try "send_command_timing", but that was not successful either.

**Comment by ktbyers** [comment]
@xvegax In general, it is the end-user's responsibility to handle any prompting that your device puts in place for commands.

There are several ways you can handle prompting like this.

Here is one way of handling this:

https://github.com/ktbyers/netmiko/blob/develop/EXAMPLES.md#handling-commands-that-prompt-expect_string
